toolkit/tools/pkgworker/pkgworker.go

```diff
@@ -80,6 +80,7 @@ func main() {
 	defines[rpm.DistTagDefine] = *distTag
 	defines[rpm.DistroReleaseVersionDefine] = *distroReleaseVersion
 	defines[rpm.DistroBuildNumberDefine] = *distroBuildNumber
+	defines[rpm.ELFHeaderPackageNoteDefine] = "1"
```

Comment:
Curious: why not make the macro set to 1 by default inside the `macros` file? This way the default value would probably be easier to find - I'm guessing most people will start looking for default macros inside RPM's various macro files and not inside our tooling.